lib/services/revanced_api.dart

```diff
@@ -12,6 +12,12 @@ import 'package:revanced_manager/models/patch.dart';
 import 'package:revanced_manager/utils/check_for_gms.dart';
 import 'package:timeago/timeago.dart';
 
+Map<String, List<dynamic>> contributors = {};
```

Comment:
I am not sure if it is traditional to abuse a map in this file for caching purposes. Instead a dedicated middleware should cache the REST api calls and IIrc there is already one. The question would be why it doesn't work.